AoE2ScenarioParser/objects/support/tile.py

```diff
@@ -94,14 +97,30 @@ def middle_location(self, other: Tile) -> tuple[float, float]:
         return (self.x + other.x) / 2, (self.y + other.y) / 2
 
     # Todo: Add tests
-    def distance_to(self, other: Tile) -> float:
+    def dist(self, other: Tile) -> float:
         """
-        Get the distance between two tiles
+        Get the euclidean distance between two tiles
 
         Args:
-            other: The tile to compare to
+            other: The tile to find the distance to
 
         Returns:
             The distance between the current and given tile
         """
         return math.sqrt((other.x - self.x) ** 2 + (other.y - self.y) ** 2)
+
+    # Todo: Add tests
+    def dist_taxi_cab(self, other: Tile) -> int:
```

Comment:
Should this be renamed to 'rectilinear distance' or 'manhatten distance'? Or something similar?
If not, the underscore between taxi and cab needs to be removed, as it's one word. 
See: https://en.wikipedia.org/wiki/Taxicab_geometry